app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if local_assigns[:model].errors.any? %>
+  <div class="alert alert-danger" role="alert">
+    <h2 class="alert__heading">There's a problem</h2>
+    <ul>
+      <% local_assigns[:model].errors.full_messages.each do |msg| %>
```

Comment:
That did cross my mind. The helper that renders this partial should never be called if the model is nil but some defensive programming here could be in order